src/lesson-4-1-demo/NumberGuesser.tsx

```diff
@@ -15,15 +15,25 @@ const getClue = (number, target) => {
   return <>Too low</>;
 };
 
-const Guess = ({ guess, targetNumber }) => {
+interface GuessProps {
```

Comment:
This is the interface for the props of the `<Guess />` component

This means that the props object for Guess should have a `guess` field that accepts a number and a `targetNumber` field that accepts a number